Scripts/script-Append.yml

```diff
@@ -0,0 +1,33 @@
+commonfields:
+  id: Append
+  version: -1
+name: Append
+script: |
+  result = demisto.args()["prefix"] + demisto.args()["suffix"]
+  res = {
+      'ContentsFormat': formats['text'],
+      'Type': entryTypes['note'],
+      'Contents': result,
+      'ReadableContentsFormat': formats['text'],
+      'HumanReadable': result,
+      'EntryContext': {demisto.args()['key']: result}
```

Comment:
script-UnzipFile.yml and there are many other python examples in the content repo